tests/common/utilities.py

```diff
@@ -1151,20 +1152,24 @@ def _paramiko_ssh(ip_address, username, passwords):
             ssh.connect(ip_address, username=username, password=password,
                         allow_agent=False, look_for_keys=False, timeout=10)
             return ssh, password
-        except paramiko.AuthenticationException:
+        except AuthenticationException:
             continue
         except Exception as e:
             logging.info("Cannot access device {} via ssh, error: {}".format(ip_address, e))
             raise e
     logging.info("Cannot access device {} via ssh, error: Password incorrect".format(ip_address))
-    raise paramiko.AuthenticationException
+    raise AuthenticationException
 
 
 def paramiko_ssh(ip_address, username, passwords):
     ssh, pwd = _paramiko_ssh(ip_address, username, passwords)
     return ssh
 
 
-def get_dut_current_passwd(ip_address, username, passwords):
-    _, pwd = _paramiko_ssh(ip_address, username, passwords)
-    return pwd
+def get_dut_current_passwd(ipv4_address, ipv6_address, username, passwords):
+    try:
+        return _paramiko_ssh(ipv4_address, username, passwords)
```

Comment:
>return [](http://example.com/codeflow?start=8&length=6)

original this function return a password string. you want to return a pair instead? Is it an unexpected behavior change?

Reply:
If you would like to keep old bahavior, suggest use a different variable name `pwd` -> `passwd`.

Reply:
updated, thanks for catching this